### DISCUSSION TITLE
🤔 交叉表带复制， 设置copyWithHeader, 得到的指标是meta中value, 而不是name

### DISCUSSION THREAD

**Discussion by feEden** [antvis/S2#2021] [Q&A]
交叉表配置了copyWithHeader： true,
得到的结果是：
```
		家具	家具	办公用品
		桌子	沙发	笔
		number	number	number
浙江省	杭州市	7789	5343	945
浙江省	绍兴市	2367	632	1304
浙江省	宁波市	3877	7234	1145
```
number 如果变成对应的显示名呢（meta中的name）

**Comment by caowenkai** [comment]
兄弟你解决了吗？这个问题这么久居然还没有答复！

**Comment by stone-lyl** [comment]
这个问题在 next 版本中已经修复了

**Reply by caowenkai** [reply]
next版本是指的 S2 2.0 next 吗？

**Reply by stone-lyl** [reply]
是的

**Comment by lijinke666** [comment]
最新 1.x 版本和 2.0 next 版本都已修复

![image](https://github.com/antvis/S2/assets/21015895/3dac9fe9-be29-4fc2-af9c-3e24475c2db5)
